### ISSUE TITLE
Make samples work

### ISSUE THREAD

**Issue by ruslanurban** [SharePoint/PnP-Provisioning-Schema#330]
**Samples should be clean and working!** *It is the guidance for new users!*
It had caused me time and a great deal of frustration to figure out why taxonomy fields modeled after the samples would not show up in the search crawled properties. I had tried a number of things to fix it: reset search indexes and re-crawled everything, forced re-indexing on lists, nothing seemed to help... Until I found articles that talk about prefixing the hidden taxonomy note fields with `TaxHTField`... I do not know the exact details how SharePoint registers the note tax field when it is created through the UI as even though the internal name does not start with the prefix, search crawling works for those fields, but when tax fields are provisioned using other methods, it requires the prefix. Must be a bug in SharePoint Search...

`WebPart` elements must have `Zone` attribute specified. It is missing in the [2018-05 full sample](https://github.com/SharePoint/PnP-Provisioning-Schema/blob/master/Samples/ProvisioningSchema-2018-05-FullSample-01.xml) (line 540), violating schema:
```xml
<pnp:WebPart Column="1" Row="1" Title="Script Editor">
```

Bottom line is, please **fix the samples**! And, **test** them before publishing! There was a number of other issues with the samples last time I tried.

**Comment by PaoloPia** [comment]
Hi,
I would suggest you exporting those kinds of fields from a live template site, rather than manually writing XML. Nevertheless, we fixed a while ago some issues related to taxonomy and to the field prefix you're talking about. I would suggest you trying again with the latest version of the engine. In case of any further issue, feel free to re-open this item. Thanks.
